taxcalc/functions.py

```diff
@@ -332,27 +336,29 @@ def EI_FICA(e00900, e02100, SS_Earnings_c, e00200, e11055, e00250, e30100,
     _sey = e00900 + e02100
     _fica_ss = max(0, FICA_ss_trt * min(SS_Earnings_c, e00200 + max(0, _sey) *
                    (1 - 0.5 * (FICA_mc_trt + FICA_ss_trt))))
-    _fica_mc = max(0, FICA_mc_trt * (e00200 + max(0, _sey)) *
-                   (1 - 0.5 * (FICA_mc_trt + FICA_ss_trt)))
+    _fica_mc = max(0, FICA_mc_trt * (e00200 + max(0, _sey) *
+                   (1 - 0.5 * (FICA_mc_trt + FICA_ss_trt))))
 
     _fica = _fica_mc + _fica_ss
 
     _setax = max(0, _fica - (FICA_ss_trt + FICA_mc_trt) * e00200)
 
-    if _setax <= 14204:
-        _seyoff = 0.5751 * _setax
-    else:
-        _seyoff = 0.5 * _setax + 10067
+    # if _setax <= 14204:
+    #    _seyoff = 0.5751 * _setax
+    # else:
+    #    _seyoff = 0.5 * _setax + 10067
+    _seyoff = 0.5 * _setax
```

Comment:
Income Tax Deduction of ½ of Self-employment Tax:
"If you have already calculated the self-employment tax, take 1/2 of the SE Tax."